From 9a699b941491775adaf41d12b106882935f11751 Mon Sep 17 00:00:00 2001
From: Lily Zhang <r58066@freescale.com>
Date: Mon, 26 Apr 2010 10:35:55 +0800
Subject: [PATCH] ENGR00122846-1 MX5: Split user and kernel space as 2G_2G

1G kernel virtual space is not insufficent for current MX53 usages.
After the discussion, agree to split kernel/user space as 2G/2G
instead of 1G/3G.

Signed-off-by: Lily Zhang <r58066@freescale.com>
---
 arch/arm/configs/imx5_defconfig |    9 ++++-----
 1 files changed, 4 insertions(+), 5 deletions(-)

diff --git a/arch/arm/configs/imx5_defconfig b/arch/arm/configs/imx5_defconfig
index dcc6881..7f43f93 100644
--- a/arch/arm/configs/imx5_defconfig
+++ b/arch/arm/configs/imx5_defconfig
@@ -1,7 +1,7 @@
 #
 # Automatically generated make config: don't edit
 # Linux kernel version: 2.6.31
-# Mon Apr 19 17:24:46 2010
+# Mon Apr 26 10:27:23 2010
 #
 CONFIG_ARM=y
 CONFIG_HAVE_PWM=y
@@ -257,10 +257,10 @@ CONFIG_TICK_ONESHOT=y
 CONFIG_NO_HZ=y
 CONFIG_HIGH_RES_TIMERS=y
 CONFIG_GENERIC_CLOCKEVENTS_BUILD=y
-CONFIG_VMSPLIT_3G=y
-# CONFIG_VMSPLIT_2G is not set
+# CONFIG_VMSPLIT_3G is not set
+CONFIG_VMSPLIT_2G=y
 # CONFIG_VMSPLIT_1G is not set
-CONFIG_PAGE_OFFSET=0xC0000000
+CONFIG_PAGE_OFFSET=0x80000000
 CONFIG_PREEMPT=y
 CONFIG_HZ=100
 CONFIG_AEABI=y
@@ -429,7 +429,6 @@ CONFIG_CAN_VCAN=y
 # CONFIG_CAN_DEV is not set
 CONFIG_CAN_DEBUG_DEVICES=y
 CONFIG_CAN_FLEXCAN=y
-CONFIG_FLEXCAN_MBM_WORD_ACCESS=y
 # CONFIG_IRDA is not set
 CONFIG_BT=y
 CONFIG_BT_L2CAP=y
-- 
1.5.4.4

